autogen.sh

```diff
@@ -30,4 +30,17 @@ git submodule update --init
 
 mkdir -p m4
 
+# ax_cxx_compile_stdcxx macro is required to check compiler option correctly
+if [ -x /usr/bin/dpkg -o -x /bin/dpkg ]; then
+  if ! dpkg -s autoconf-archive > /dev/null; then
+    echo "ERROR: autoconf-archive package is not installed yet."
+    exit 1
+  fi
+elif [ -x /usr/bin/rpm -o -x /bin/rpm ]; then
+  if ! rpm -q autoconf-archive > /dev/null; then
+    echo "ERROR: autoconf-archive package is not installed yet."
+    exit 1
+  fi
+fi
```

Comment:
What if `[ -x /usr/bin/dpkg -o -x /bin/dpkg ]` is false and `[ -x /usr/bin/rpm -o -x /bin/rpm ]` is false?

Reply:
Ah, surely such a case is out-of-scope in above commit.
It should be fixed.

Reply:
I've fixed to check about existence of macro file as a fallback.